### PR TITLE
5802 Add Pagination Params on GET Areas response

### PR DESCRIPTION
**Pull request by PauloPortugal**
### What

[5802 Add Pagination Params on GET Areas response](https://github.com/ONSdigital/dp-api-clients-go/commit/6729d44eb4bb2635304fe681e571c4d10f1b5469)

### How to review

Sense Check and review this [PR](https://github.com/ONSdigital/dp-population-types-api/pull/63).

### Who can review

Any ONS developer

### CODE REVIEW

**Review by j-s-rawat — DISMISSED** [review]
LGTM

File: cantabular/dimensions.go [hunk 1]
```diff
@@ -260,7 +267,18 @@ func (c *Client) GetParents(ctx context.Context, req GetParentsRequest) (*GetPar
 
 	// last item is guaranteed to be provided variable, only return parents
 	edges := resp.Data.Dataset.Variables.Edges[0].Node.IsSourceOf.Edges
-	resp.Data.Dataset.Variables.Edges[0].Node.IsSourceOf.Edges = edges[:len(edges)-1]
+	for i, v := range edges {
```
[the remark this answers is not in the document]
Comment (PauloPortugal) [reply cantabular/dimensions.go:270]: Previously we had


```
resp.Data.Dataset.Variables.Edges[0].Node.IsSourceOf.Edges = edges[:len(edges)-1]
```

Yesterday spoke with Fran and she told me that one area-type could have more than one parent, so if we want to perform a GraphQL query with the `limit` parameter we need to account for the fact that potentially we only have one item and we cannot confidently  ensure that the last item of a `limit: 2` is the provided variable.

Therefore we need to iterate through the edges array and remove the item that is the provided variable.

[the remark this answers is not in the document]
Comment (PauloPortugal) [reply cantabular/dimensions.go:270]: Another way would be to visit the last element of the array and check if it is the provided variable.

**Review by j-s-rawat — COMMENTED** [review]
LGTM

**Review by j-s-rawat — APPROVED** [review]

**Pull request merged**